tests/test_partial.py

```diff
@@ -28,16 +30,24 @@
 Z = da.from_array(z, chunks=(2, 2))
 
 
-def test_fit():
+@pytest.mark.parametrize("x_,y_,z_", [(x, y, z), (X, Y, Z)])
+def test_fit(x_, y_, z_):
     with dask.config.set(scheduler='single-threaded'):
         sgd = SGDClassifier(max_iter=5)
-
-        sgd = fit(sgd, X, Y, classes=np.array([-1, 0, 1]))
+        sgd = fit(sgd, x_, y_, classes=np.array([-1, 0, 1]))
 
         sol = sgd.predict(z)
-        result = predict(sgd, Z)
-        assert result.chunks == ((2, 2),)
-        assert result.compute().tolist() == sol.tolist()
+        result = predict(sgd, z_)
+
+        if isinstance(z_, da.Array):
+            assert result.chunks == ((2, 2),)
+            assert isinstance(result, da.Array)
+            assert result.compute().tolist() == sol.tolist()
```

Comment:
FYI, we'd typically use `dask.array.utils.assert_eq` (I know you didn't write this test).

Reply:
Good to know. I thought that seemed clunky.